python_modules/dagster/dagster/_core/definitions/sensor_definition.py

```diff
@@ -618,6 +734,7 @@ def build_sensor_context(
     repository_name: Optional[str] = None,
     repository_def: Optional["RepositoryDefinition"] = None,
     sensor_name: Optional[str] = None,
+    resource_defs: Optional[Mapping[str, "ResourceDefinition"]] = None,
```

Comment:
document this in the docblock and specify whether it is this resource_defs or the Definitions resources that take precedences